### ISSUE TITLE
Broken in master for several months

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-black-box-study#62]
This sim has been broken in master for several months as we pushed to get CCK DC ready for publication.  @ariel-phet can you please help prioritize and schedule cleanup for the Black Box?  I'd guess several days to get it working in master again, plus more time to go through open issues and address them.  May be best to do after we have stable shas for CCK so we don't slow down its release.

**Comment by samreid** [comment]
Note also https://github.com/phetsims/axon/issues/140#issuecomment-330317752 is broken for Black Box Study.

**Comment by ariel-phet** [comment]
@samreid I think https://github.com/phetsims/axon/issues/140#issuecomment-330317752 should be fixed sooner than later if that can be done separately from fixing the entire black box sim.

I also wonder if we should rename this repo simply to "black-box" as the suffix instead of "black-box-study" since I believe Kathy is interested in actually producing a published black-box. Or perhaps we need a separate repo for black-box 

As for priority, CCK being published should be the focus. So lets at least get that to RC and be confident it is fairly stable before working on this further clean-up.

**Comment by samreid** [comment]
https://github.com/phetsims/axon/issues/140#issuecomment-330317752 is fixed for Circuit Construction Kit: DC and Circuit Construction Kit: DC - Virtual Lab.  It cannot be tested for Circuit Construction Kit Black Box Study because it is not currently launchable.  So we would need to get Circuit Construction Kit Black Box Study running again before we can work on the axon issue (or even determine if it is a problem here).  Reassigned to @ariel-phet for discussion/clarification.

**Comment by ariel-phet** [comment]
@samreid I thought that might be the case (that to fix that issue the sim would need to be running).

Let's continue to keep the focus on getting CCK out. Certainly you could begin some clean up of Black Box while CCK is currently in dev testing, with the priority to be cleaning up Black Box enough to get it launchable.

**Comment by samreid** [comment]
Sounds good, I'll put this on my to-do list, though if I run into issues that require non-trivial changes to the CCK common repo, I'll want to wait until we have safe shas.

**Comment by samreid** [comment]
After that commit, screen 1 is working again.

**Comment by samreid** [comment]
@kathy-phet said this is low priority.  We have a stable version we can use and shouldn't work on this further without discussion.

**Comment by KatieWoe** [comment]
Visual example:
![blackboxwrong](https://user-images.githubusercontent.com/41024075/75714830-a5ca6080-5c89-11ea-9efe-3ec3d12b180e.png)
